### ISSUE TITLE
Batch support

### ISSUE THREAD

**Issue by jorisguerin** [Raj-08/tensorflow-object-contour-detection#10]
Hello, thanks for the great code!

Is there a reason for treating all images independently and not making batches for training?

Cheers

**Comment by rubeea** [comment]
Why is batch normalization applied after each conv layer of the encoder and decoder when the training is based on independent images and not batches?
